queries/r/textobjects.scm

```diff
@@ -79,44 +67,44 @@
   .
   (_) @parameter.inner
   .
-  ","? @_end)
+  (comma)? @_end)
   (#make-range! "parameter.outer" @parameter.inner @_end))
 
 ; number
 (float) @number.inner
 
 ; assignment
-(left_assignment
+("<-"
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(left_assignment
+("<-"
   name: (_) @assignment.inner)
 
-(right_assignment
+("->"
   value: (_) @assignment.inner @assignment.lhs
   name: (_) @assignment.rhs) @assignment.outer
 
-(right_assignment
+("->"
   name: (_) @assignment.inner)
 
-(equals_assignment
+((comma)
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(equals_assignment
+((comma)
   name: (_) @assignment.inner)
 
-(super_assignment
+("<<-"
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(super_assignment
+("<<-"
   name: (_) @assignment.inner)
 
-(super_right_assignment
+("->>"
   value: (_) @assignment.inner @assignment.lhs
   name: (_) @assignment.rhs) @assignment.outer
 
-(super_right_assignment
-  name: (_) @assignment.inner)
+(binary_operator)
+  name: (_) @assignment.inner
```

Comment:
```suggestion
(binary_operator
  name: (_) @assignment.inner)
```

Reply:
Then this is not exactly what you want then :). Please remove it if it's unnecessary

Reply:
You still need to remove these 2 lines. They don't make sense.